libcudacxx/test/libcudacxx/cuda/ptx/ptx.cp.async.bulk.multicast.compile.pass.cpp

```diff
@@ -0,0 +1,49 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+// UNSUPPORTED: libcpp-has-no-threads
+
+// UNSUPPORTED: nvcc-11
+// XFAIL: !pre-sm-90 && pre-sm-90a
```

Comment:
This test is expected to succeed only on sm90a. So couldn't we make this `!pre-sm90a`?

Reply:
The test does not fail if we PTX ISA 8.0 is not exposed, so on 11.8 we can test against SM90 and it wont fail

Reply:
Also that test wont fail pre-sm-90 because we do NV_IF_TARGET it out

Reply:
Good point. LGTM then!